internal/naming/annotations.go

```diff
@@ -68,4 +68,8 @@ const (
 	// to a cluster without backups. As usual with the operator, we do not
 	// touch cloud-based backups.
 	AuthorizeBackupRemovalAnnotation = annotationPrefix + "authorizeBackupRemoval"
+
+	// Used from Kubernetes v1.21+ to define a default container used when the
+	// `-c` flag is not passed
+	DefaultContainerLabel = "kubectl.kubernetes.io/default-container"
```

Comment:
⛏️ This is an annotation, but it's named `Label` here.

Could link to https://docs.k8s.io/reference/labels-annotations-taints#kubectl-kubernetes-io-default-container.